src/components/LoadingSkeleton.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<div>
 		<div v-for="i in numberOfLines" :key="i" class="item-list__entry">
-			<div
+			<div v-if="withAvatar"
```

Comment:
this smells unrelated :D

Reply:
It is but it was missing from LoadingSkeleton, I'll create a separate Pr for it

Reply:
#8674